deploy/olm-catalog/mobile-developer-console-operator/mobile-developer-console-operator.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: integreatly-mobile-developer-console
+channels:
+  - name: integreatly
+    currentCSV: mobile-developer-console-operator.v0.2.6
```

Comment:
Does the CSV have to match the operator image version or can these be different? I suppose the question is, do we do a version release of the operator in github and then in another pr do the CSV file generation?

Reply:
I reckon every time the operator gets updated another CSV file needs to be generated

Reply:
We should ideally create the CSV as part of the release I'd think. E.g. a commit that updates the version.go & CHANGELOG.md & has the new CSV. This would be the commit that is tagged & that the image is built from.